xbmc/main/posix/MessagePrinter.h

```diff
@@ -0,0 +1,53 @@
+/*
+*      Copyright (C) 2005-2015 Team XBMC
+*      http://xbmc.org
+*
+*  This Program is free software; you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 2, or (at your option)
+*  any later version.
+*
+*  This Program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with XBMC; see the file COPYING.  If not, see
+*  <http://www.gnu.org/licenses/>.
+*
+*/
+
+#include <string>
+#include <vector>
+#include <utility>
+
+class CMessagePrinter
+{
+public:
+
+  /*! \brief Display a normal message to the user during startup
+  *
+  * \param[in] message  message to display
+  */
+  void DisplayMessage(const std::string& message);
```

Comment:
First design used an interface and inherited classes but i scrapped that they got left as non static, I'll change this.